### DISCUSSION TITLE
Netlify - This function has crashed

### DISCUSSION THREAD

**Discussion by monkemedia** [nuxt/nuxt#15948] [Questions]
Has there been any updates to any dependencies recently? My production website www.marblesofmumbles.com will no longer show. It is currently hosted on Netlify and after a rebuild I am getting the following error:

```
Runtime.ImportModuleError - Error: Cannot find module 'server' Require stack: - /var/runtime/index.mjs

Stack trace
Runtime.ImportModuleError: Error: Cannot find module 'server'
Require stack:
- /var/runtime/index.mjs
    at _loadUserApp (file:///var/runtime/index.mjs:1000:17)
    at async Object.UserFunction.js.module.exports.load (file:///var/runtime/index.mjs:1035:21)
    at async start (file:///var/runtime/index.mjs:1200:23)
    at async file:///var/runtime/index.mjs:1206:1
```

Please note the website works locally and also works locally when i do a `build`

**Comment by gjvengelen** [comment]
There are two work arounds which can be applied while we wait for a solution.

1. You can switch to netlify-edge preset to use Edge Functions instead of Functions to render Nuxt content.
2. You can lookup a deployment from before 11-1-2023 14:00 and click Publish deploy in Netlify. At least your site will be up again.

**Reply by monkemedia** [reply]
I ended up doing option 2 which worked for me. Thanks @gjvengelen

**Comment by fro-profesional** [comment]
Issues was fixed https://github.com/nuxt/framework/issues/10050 thanks nuxt and netlify team <3 

-- This discussions should be closed ?
